### ISSUE TITLE
Negative Operating Costs, Maintenance Costs

### ISSUE THREAD

**Issue by AvianGeneticist** [endless-sky/endless-sky#6406]
### Problem Description

I think it'd be really cool if there were outfits that allowed the player to make money; they'd have to be saddled with hefty drawbacks, of course, like huge sizes, power/heat levels, crew requirements, et cetera, but it'd be really cool to have such an option for later-game players looking to offset their daily credit drain.

That said, art and ideation is hard, so I'm not demanding that outfits be made for the base game; I'm just requesting that maintenance and operating cost values be adjusted so they **can** be set to negative, so they give the player credits instead of taking them away at a daily rate.

Honestly, I'd try and implement this change myself if I knew how to code.

### Related Issue Links

Couldn't find any posts with this specific request. If it's been sent before, I apologize.

### Desired Solution

The ability to make the values "Maintenance Costs" and "Operating Costs" negative, and have them give the player credits instead of taking them away per day.

### Alternative Approaches

Adding a new value that gives the player the number specified of credits per day (maybe called "Gross Profit" or something of the sort).

**Comment by Zitchas** [comment]
I like the idea of being able to make things that pay back. Whether it is investments, or renting out ship power to do something (crypto-currency mining in the 31st century anyone?), I think it's an option that makes sense to have.

**Comment by Ferociousfeind** [comment]
There are a couple of attributes that can go negative (can't remember any off the top of my head) but MOST specifically-delineated attributes (and ALL attributes that are secretly another attribute but negative effect) should be on that whitelist.

**Comment by AvianGeneticist** [comment]
When I tried it for an outfit I'm coding, the game got mad because the outfit would've made one of my ship's attributes negative.
I made sure it wasn't any other quantity (not hard, since it only used outfit and cargo space); it's definitely the operating cost value that was the issue.

**Comment by Beanzilla** [comment]
Sounds almost like adding a salary would be the solution.

> Try my own mod/plugin, [Player Rewards](https://github.com/Beanzilla/PlayerRewards). (Place the repo/directory into your plugins directory)

**Comment by petervdmeer** [comment]
I created a sample implementation (which got merged), but then we concluded that separate attributes are nicer than re-using the existing `"operating costs"` and `"maintenance costs"` attributes.

So what we need for (re)implementing this feature is input from content-creators to get two good names for the new attributes (and the tool-tip texts that go with them). For example something like:
- `"operating income"` => `"The amount of credits acquired each day that this ship/outfit is operational."`
- `"owner income"` => `"The amount of credits acquired each day that this ship/outfit is owned."`

tehhowch also created some suggestions [here](https://github.com/endless-sky/endless-sky/pull/6425#issuecomment-1000290620).
